src/Appwrite/Utopia/Response/Model/Collection.php

```diff
@@ -60,11 +53,11 @@ public function __construct()
                 'default' => true,
                 'example' => false,
             ])
-            ->addRule('permission', [
-                'type' => self::TYPE_STRING,
-                'description' => 'Collection permission model. Possible values: `document` or `collection`',
+            ->addRule('documentSecurity', [
```

Comment:
Just for the attribute name or for the UI as well? `security` by itself sounds like it would be dangerous to disable, even though when disabled collection/bucket permissions would still apply